.claude/skills/e2e-testing/SKILL.md

```
@@ -316,3 +316,9 @@ test.describe("Auth Protection: [対象エンドポイント]", () => {
 - **実装を確認せずにテストを書かない**
 - **「こうあるべき」という推測でテストを書かない**
 - **MVP範囲外の機能をテストしない**
+
+## 参照ドキュメント
```

markdownlint-cli2 (0.21.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)